glib/src/functions.rs

```diff
@@ -231,7 +231,7 @@ pub fn unix_open_pipe(flags: i32) -> Result<(RawFd, RawFd), Error> {
     unsafe {
         let mut fds = [0, 2];
         let mut error = ptr::null_mut();
-        let _ = ffi::g_unix_open_pipe(fds.as_mut_ptr(), flags, &mut error);
+        let _ = ffi::g_unix_open_pipe(&mut fds, flags, &mut error);
```

Comment:
what was the reason of this change? could be split in a separate commit

Reply:
The gir file changed so the parameter of the -sys function changed from `*mut i32` to `[i32; 2]`

Reply:
ah right